gef.py

```diff
@@ -8893,12 +8893,35 @@ class VMMapCommand(GenericCommand):
     _example_ = f"{_cmdline_} libc"
 
     @only_if_gdb_running
-    def do_invoke(self, argv: List[str]) -> None:
+    @parse_arguments({"unknown_types": [""]}, {("--addr", "-a"): [""], ("--name", "-n"): [""]})
+    def do_invoke(self, _: List[str], **kwargs: Any) -> None:
+        args : argparse.Namespace = kwargs["arguments"]
         vmmap = gef.memory.maps
         if not vmmap:
             err("No address mapping information found")
             return
 
+        addrs: Dict[str, int] = {x: parse_address(x) for x in args.addr}
+        names: List[str] = [x for x in args.name]
+
+        for arg in args.unknown_types:
+            if not arg:
```

Comment:
why would this ever hit? Is this because the default is a list with a single empty string? Can we just make the default an empty list?

Reply:
In the `parse_args` decorator, we must provide something like `[""]` for it to know that it is a list of `str`. Maybe the decorator should be modified in order to remove the first element of the array if it is empty, but I guess it should be done in another PR